src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.form-field {
+  display: flex;
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+}
+
+.form-field__small-bar {
```

Comment:
you use `small` only here
in all other places, you use `big` to add overwrite styles, Be consistent - read [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 2, remove unnecessary classes in the HTML after all